Can't compile on manjaro linux

Can't compile due to a version issue in cmake.

[darkstar@wopr build]$ cmake ../
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


[darkstar@wopr build]$ cmake --version
cmake version 4.0.3-dirty

CMake suite maintained and supported by Kitware (kitware.com/cmake).

Playing more i changed the minimun version in CMakeLists to 3.5 but ... Errors coming.

[darkstar@wopr OpenNFS-0.3]$ cd build
[darkstar@wopr build]$ cmake ..
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


-- The C compiler identification is GNU 15.1.1
-- The CXX compiler identification is GNU 15.1.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:145 (add_subdirectory):
  The source directory

    /home/darkstar/Documentos/src/OpenNFS-0.3/lib/boost-cmake

  does not contain a CMakeLists.txt file.


CMake Error at CMakeLists.txt:150 (add_subdirectory):
  The source directory

    /home/darkstar/Documentos/src/OpenNFS-0.3/lib/g3log

  does not contain a CMakeLists.txt file.


CMake Error at CMakeLists.txt:164 (add_subdirectory):
  The source directory

    /home/darkstar/Documentos/src/OpenNFS-0.3/lib/bullet3

  does not contain a CMakeLists.txt file.


-- Found OpenGL: /usr/lib/libOpenGL.so
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:227 (message):
  Could NOT find Vulkan (missing: Vulkan_INCLUDE_DIR) (found version "")
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:591 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake/Modules/FindVulkan.cmake:594 (find_package_handle_standard_args)
  CMakeLists.txt:178 (find_package)


-- Configuring incomplete, errors occurred!
[darkstar@wopr build]$ cmake ..
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.


CMake Error at CMakeLists.txt:145 (add_subdirectory):
  The source directory

    /home/darkstar/Documentos/src/OpenNFS-0.3/lib/boost-cmake

  does not contain a CMakeLists.txt file.


CMake Error at CMakeLists.txt:150 (add_subdirectory):
  The source directory

    /home/darkstar/Documentos/src/OpenNFS-0.3/lib/g3log

  does not contain a CMakeLists.txt file.


CMake Error at CMakeLists.txt:164 (add_subdirectory):
  The source directory

    /home/darkstar/Documentos/src/OpenNFS-0.3/lib/bullet3

  does not contain a CMakeLists.txt file.


-- Found Vulkan: /lib/libvulkan.so (found version "1.4.313") found components: glslc glslangValidator
VULKAN FOUND
The following GLSL shaders will be compiled to SPIRV:
- /home/darkstar/Documentos/src/OpenNFS-0.3/shaders/vk/shader.frag
- /home/darkstar/Documentos/src/OpenNFS-0.3/shaders/vk/shader.vert
CMake Error at CMakeLists.txt:188 (add_subdirectory):
  The source directory

    /home/darkstar/Documentos/src/OpenNFS-0.3/lib/glfw

  does not contain a CMakeLists.txt file.


-- Configuring incomplete, errors occurred!

--- a/.gitmodules
+++ b/.gitmodules
@@ -33,4 +33,4 @@
 	url = https://github.com/google/googletest.git
 [submodule "lib/LibOpenNFS"]
 	path = lib/LibOpenNFS
-	url = git@github.com:OpenNFS/LibOpenNFS.git
+	url = https://github.com/OpenNFS/LibOpenNFS.git

